static/src/javascripts/projects/common/modules/crosswords/anagram-helper/main.js

```diff
@@ -100,9 +100,13 @@ const AnagramHelper = React.createClass({
             this.props.entries,
             this.props.focussedEntry
         );
-        const entries = cells.map(
-            coords => this.props.grid[coords.x][coords.y]
-        );
+        const entries = cells.map(coords => {
+            const entry = this.props.grid[coords.x][coords.y];
+
+            entry.key = `${coords.x},${coords.y}`;
+
+            return entry;
+        });
```

Comment:
I wonder if we can one-line this badboy?

```
const entries = cells.map(coords => Object.assign({}, this.props.grid[coords.x][coords.y], { key: `${coords.x},${coords.y}` }));
```

💥 

Actually, perhaps this is indecipherable!